.github/actions/teardown-rocm/action.yml

```diff
@@ -17,3 +17,7 @@ runs:
     - name: Runner diskspace health check
       uses: ./.github/actions/diskspace-cleanup
       if: always()
+    - name: Chown workspace back to pytorchci (Fallout of PR 122922)
+      if: always()
+      run: |
+        sudo chown -R pytorchci ${GITHUB_WORKSPACE}
```

Comment:
@clee2000 Is there a need to add a `| true` after this command to ensure it always exits with a 0 code? I assume `${GITHUB_WORKSPACE}` should always exist at this point and we **do** want to detect any error when trying the chown?

Reply:
You could set `continue-on-error` to true to not failing this step, i.e. https://github.com/pytorch/pytorch/blob/ae6f8d923ceb032d3e933507b071284ba7ec218d/.github/workflows/unstable.yml#L21

Reply:
@huydhn I'm concerned if this might lead to this step silently failing but we won't notice it on the HUD until it's probably too late? For eg. if the disk cleanup or chown is failing, we do want to detect an error? Thoughts?